infrastructure/helm/tigerbeetle/templates/configmap.nginx.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-nginx-proxy
+data:
+  # property-like keys; each key maps to a simple value
+  nginx-prestart.sh: |
+    #!/bin/sh
+    sleep 600
```

Comment:
This sleep is long. It is designed to be long enough to cover the entire period of rolling all of the replicas, because otherwise hostname resolution can fail after an nginx is up, or while it is coming up, causing a kind of round-robin failure that can bring down the whole cluster.

Reply:
No longer needs a sleep; resolver directive instead.